packages/contracts/src/spaces/facets/account/AppAccountBase.sol

```
@@ -208,6 +209,19 @@ abstract contract AppAccountBase is
         return AppAccountStorage.getLayout().installedApps.values();
     }
 
+    function _isAppExecuting(address app) internal view returns (bool) {
+        bytes32 currentExecutionId = ExecutorStorage.getExecutionId();
+        bytes32 targetId = ExecutorStorage.getTargetExecutionId(app);
+
+        if (currentExecutionId == bytes32(0) || targetId == bytes32(0)) return false;
```

```suggestion
        bytes32 currentExecutionId = ExecutorStorage.getExecutionId();
        if (currentExecutionId == bytes32(0)) return false;

        bytes32 targetId = ExecutorStorage.getTargetExecutionId(app);
        if (targetId == bytes32(0)) return false;
```
nit: fail fast and save one `sload`.